hub/components/RealmHeader/index.tsx

```diff
@@ -84,25 +84,7 @@ async function importJupiter() {
     }
   });
 
-  const css = new Promise((res, rej) => {
-    const existing = document.getElementById(
-      'jupiter-load-styles',
-    ) as HTMLLinkElement | null;
-
-    if (existing) {
-      res({});
-    } else {
-      const el = document.createElement('link');
-      el.onload = res;
-      el.onerror = rej;
-      el.id = 'jupiter-load-styles';
-      el.rel = 'stylesheet';
-      el.href = 'https://terminal.jup.ag/main.css';
-      document.head.append(el);
-    }
-  });
-
-  return Promise.all([script, css]).then(() => {
+  return Promise.all([script]).then(() => {
```

Comment:
nit: no need for `Promise.all` anymore now that there is only one promise

Reply:
this is done!

Reply:
also, I've noticed that user's aren't actually auto connected on refresh some times, I've also pushed a conditional that only does passThrough when it is connected.
```tsx
passThroughWallet: connected ? wallet : undefined,
```